lib/Db/FaceMapper.php

```diff
@@ -131,6 +131,51 @@ public function findFacesFromPerson(string $userId, int $personId, int $model, $
 		return $faces;
 	}
 
+	public function findRepresentativeFromPerson(string $userId, int $personId, float $sensitivity, int $model) {
```

Comment:
this would be handy to precompute during cluster creation? It is not slower than cluster creation itself, and can dramatically speed up this logic.

Reply:
I think you already have all these distances in that logic, so maybe there is not even a need to call `dlib_vector_length` two times (I am not sure about this, though)

Reply:
Yes .. As I said, it can be done and would greatly improve the calculations here ..

If you want I implement it.. Only need to add a column to oc_facerecog_persons. This would be a representative_face? Or directly the confidence of this face?

Mm.. Note that representative_face may seem to be the face to show in the group... But they would be incompatible. :confused: